consensus/reactor_test.go

```diff
@@ -65,6 +65,12 @@ func TestValidatorSetChanges(t *testing.T) {
 	nPeers := 8
 	nVals := 4
 	css := randConsensusNetWithPeers(nVals, nPeers, "consensus_val_set_changes_test", newMockTickerFunc(true))
+
+	// otherwise, the test is failing with timeout error
```

Comment:
Ok, I see now. It's either that or change onlyOnce to false: `newMockTickerFunc(false)` (otherwise, ticker only fires once and cannot make it to a new height >= 2).